src/js/components/Diagram/StyledDiagram.js

```diff
@@ -1,12 +1,80 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 import { defaultProps } from '../../default-props';
+import { animationObjectStyle } from '../../utils/animations';
+
+const animationItemStyle = (animationType, theme) => {
+  if (typeof animationType === 'string') {
+    return animationObjectStyle({ type: animationType }, theme);
+  }
+  if (typeof animationType === 'object') {
+    return animationObjectStyle(animationType, theme);
+  }
+  if (typeof animationType === 'boolean') {
+    return animationObjectStyle({ type: 'draw' }, theme);
+  }
+  return '';
+};
+
+const availableAnimations = [true, 'draw', 'pulse'];
+
+const animationStyle = props => {
+  const animationType = props.animation.type || props.animation;
+
+  if (!availableAnimations.includes(animationType)) {
```

Comment:
We don't frequently return messages -- nothing inherently wrong here but just breaks a little form our typical pattern. Wondering instead if `animationStyle` should only be called if `availableAnimation.includes(animationType)`. This check should go below wherever animationStyle is being called

Reply:
Great point and thanks for the tip, I definitely agree with this, trying it out now.

Reply:
For `animationStyle` I implemented it when we're calling `${props => props.animation && ...}` but as for `connectionStyle` I did take out the return message and left it as an empty string but left the `if statement` since the check has to happen inside the `connections.map`. Let me know your thoughts.